index.html

```diff
@@ -17,6 +17,7 @@ <h1>Headphones recommended</h1>
   <canvas class="background"></canvas>
   <div class="greeting main-greeting">Have a peaceful 2021</div>
   <div class="greeting secondary-greeting">from Nishant, Varsha & Mahi</div>
+  <button onclick="toggleAudio()" type="button" class="mute-unmute-toggle">Mute</button>
```

Comment:
We do not mix content and behaviour. add the click event inside `index.js` once the document is loaded.